src/main/windows.ts

```diff
@@ -79,6 +88,15 @@ export const setupAppWindows = () => {
     }
   });
 
+  globalShortcut.register('CommandOrControl+Shift+I', () => {
```

Comment:
We need to choose a different shortcut here. `CommandOrControl+Shift+I` is the default for toggling the developer tools and now it's not possible anymore to toggle developer tools in happ windows.

Reply:
do you have a suggestion?

Reply:
I'm not sure either...something that is not used by other things ideally (if that's possible at all) and ideally also somewhat ergonomic to press. On Windows/Linux I like `Super+something` combinations but I'm not sure that works on mac.

Reply:
We could take `CommandOrControl+Shift+L` for now?